src/app/clients/ClientsPage.tsx

```diff
@@ -178,6 +186,11 @@ const ClientsPage: React.FC<Props> = (props) => {
                     <ExpandedClientDetails parcelId={selectedParcelId} />
                 </Suspense>
             </Modal>
+            <CenterComponent>
+                <Button variant="contained" onClick={onClickAddClients}>
```

Comment:
Does this need to be separate? Just bc it's a short function anyway and could just be put inline.